spec/active_record/pg_enum_spec.rb

```diff
@@ -45,5 +45,15 @@ class TestTable < ActiveRecord::Base
       expect(TestTable).to respond_to :foo_types
       expect(TestTable.foo_types).to match({ "bar" => "bar", "baz" => "baz" })
     end
+
+    it "passes any additional options to ::enum", version: ">= 5.0" do
```

Comment:
Enum options [were first added in Rails 5.0](https://guides.rubyonrails.org/5_0_release_notes.html#active-record-notable-changes)